exports/context.tf

```
@@ -68,6 +70,8 @@ variable "context" {
     regex_replace_chars = null
     label_order         = []
     id_length_limit     = null
+    label_value_case    = null
+    label_key_case      = null
```

Traditionally and alphabetically, key comes before value
```suggestion
    label_key_case      = null
    label_value_case    = null
```